docs/conf.py

```diff
@@ -5,6 +5,8 @@
 
 # -- Project information -----------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+import sys
```

Comment:
Does `make -C docs livehtml` work when run from the root of the repo?

Reply:
Yes